src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -115,7 +124,7 @@ export const SavedSearchAlertListItemFragmentContainer = createFragmentContainer
         href
         title: displayName(only: [artistIDs])
         subtitle: displayName(except: [artistIDs])
-        artworksConnection(first: 1) {
+        artworksConnection(first: 10) {
```

Comment:
👀

Reply:
GraphQL was complaining
` Expected all fields on the same parent with the name or alias `artworksConnection` to have the same argument values after applying fragment arguments.`
Do you know if we could keep using `artworksConnection(first: 1)` in SavedSearchAlertListItem.tsx?

Reply:
That makes sense. If there is another fragment spread in the query for the same `artworksConnection`, then  align the parameters need to be aligned.

If you would need to keep the parameter `first: `), you could rename the field. However, this would result in fetching the connection twice, and I don't think we should do it in this case.

```suggestion
        smallArtworkConnection: artworksConnection(first: 1) {
```

Reply:
>you could rename the field. However, this would result in fetching the connection twice
+++

I would keep it as it is then. What do you think?

Reply:
Yes, let's keep it as it is 👍